docs/guide/self-hosting.md

```
@@ -90,8 +90,74 @@ browser → VPS (public URL) → tunnel → relay Mac (office)
+rathole --server /etc/rathole-server.toml
+```
+
+#### 2. Set up Caddy for HTTPS
+
+Caddy handles TLS automatically — no certbot needed.
+
+```sh
+sudo apt install -y caddy
+```
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)